integration/index.ts

```diff
@@ -64,8 +72,3 @@ expect.extend({
     return { pass: true, message: () => `expected ${received} not to equal ${argument}` };
   },
 });
-
-// Sleep a bit before each test to avoid being rate-limited.
```

Comment:
I think we should still have a short timeout because it did reduce 429's by a huge amount. With your solution it looks like we still introduce a delay between retries, so why not have the delay in the first place and avoid making 2x or more the requests.

Reply:
It's using exponential delay that increases on each retry and first time it's only 100ms. I believe having on-demand retry serves better purpose because most of the tests will run smoothly and only some of them will experience retries (just axios request retries, not whole tests) - from testing locally it only actually happens on few tests in skydb data race and upload-download suites, probably those are the request heavy ones.

Reply:
Sounds good. 👍